### DISCUSSION TITLE
1101-10E-SP76-16S also confirmed working

### DISCUSSION THREAD

**Discussion by tobox** [syssi/esphome-seplos-bms#42] [Show and tell]
I can confirm that your code works well with my unit, although it was not listed in your supported devices section.

![grafik](https://user-images.githubusercontent.com/4719666/223078423-1cf632b1-ae0d-4c1e-afcd-3e4e9086ece1.png)

**Comment by syssi** [comment]
Thanks for your feedback! I've extended the list of supported devices.

**Comment by saviothecnic** [comment]
> I can confirm that your code works well with my unit, although it was not listed in your supported devices section.
> 
> ![grafik](https://user-images.githubusercontent.com/4719666/223078423-1cf632b1-ae0d-4c1e-afcd-3e4e9086ece1.png)

Dude it looks the same as my Seplos 10E V2.0 device
Can you tell me what firmware you uploaded to your yaml
and your ESP module configuration?

MY Firmware 
![image](https://github.com/syssi/esphome-seplos-bms/assets/6697807/c6822978-4880-446f-a63c-258d6cbbd484)
